packages/datagateway-dataview/src/page/pageRouting.component.tsx

```diff
@@ -317,8 +317,21 @@ class PageRouting extends React.PureComponent<PageRoutingProps> {
         />
 
         {/* My Data routes */}
-        <Route path={paths.myData.dls} component={DLSMyDataTable} />
-        <Route path={paths.myData.isis} component={ISISMyDataTable} />
+        <Route exact path={paths.myData.dls}>
+          {localStorage.getItem('autoLogin') === 'true' ? (
```

Comment:
I made this similar comment to Joel so it might be worth looking at his "anonymous" PR, but I would prefer that this checks for anonymous as theoretically you can login as anonymous without being autoLogged in.

Reply:
I have implemented this yesterday, now I'm working on changing the testing, so it works for changes (anonymous login)